bios/brianatssat.md

```diff
@@ -0,0 +1,4 @@
+## brianatssat
+- IT Director at SSAT
+- Wants to learn more about Git
```

Comment:
How about we replace this with something else?  It is a given that you want to know Git since you are in a Git class.